### PR TITLE
[PKG-2]: Adds pgvector packages.

### PR DESCRIPTION
**Pull request by maqeel75**

### CODE REVIEW

File: pgvector/rules [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#
+#override_dh_auto_install:
+#	+pg_buildext install build-%v postgresql-%v-pgvector
+#
+#override_dh_installdocs:
+#	dh_installdocs --all README.*
+#
+#override_dh_auto_clean:
+#	+pg_buildext clean build-%v
+#
```
Comment (EvgeniyPatlan) [inline pgvector/rules:31]: do we need all these commented lines?

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: pgvector/control [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Source: percona-pgvector
+Section: database
+Priority: optional
+Maintainer: Debian PostgreSQL Maintainers <team+postgresql@tracker.debian.org>
```
[the remark this answers is not in the document]
Comment (maqeel75) [reply pgvector/control]: Fixed.

File: pgvector/control [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Uploaders:
+ Bradford D. Boyle <bradford.d.boyle@gmail.com>,
```
[the remark this answers is not in the document]
Comment (maqeel75) [reply pgvector/control]: Fixed.

File: pgvector/rules (hunk 1 above)
Comment (maqeel75) [reply pgvector/rules]: This is how the community repository had this file. Removed all commented lines.

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**